### DISCUSSION TITLE
Microsoft as a provider in Email Inboxes - not exists in version 2.17.1

### DISCUSSION THREAD

**Discussion by Naomi1900** [chatwoot/chatwoot#7176] [Self hosted]
I installed Chatwoot  version 2.17, and I don't see the option to create email inbox to Microsoft provider.
I don't see this option:
https://www.chatwoot.com/docs/product/channels/email/microsoft/create-channel

Do I need to configure something to have the option to create Microsoft email inbox?
Thanks

**Comment by sojan-official** [comment]
https://www.chatwoot.com/docs/self-hosted/configuration/features/email-channel/azure-app-setup
